airflow/utils/operator_helpers.py

```diff
@@ -260,6 +260,11 @@ def _run():
         for metadata in _run():
             if isinstance(metadata, Metadata):
                 self.outlet_events[metadata.uri].extra.update(metadata.extra)
+
+                if metadata.alias:
+                    for alias in metadata.alias:
+                        self.outlet_events[alias].add(metadata.uri, extra=metadata.extra)
```

Comment:
If this is always a list, we can remove the `if` check.

Reply:
yep, after the rewrite, i just removed this check